R/convertFromBIOM.R

```diff
@@ -69,31 +84,30 @@
 #'                     artifact.rm = TRUE)
 NULL
 
-#' @rdname makeTreeSEFromBiom
-#'
+#' \code{importBIOM} loads a BIOM file and creates a TreeSE from the BIOM object
+#' contained in the loaded file.
+#' @rdname convert
 #' @export
 importBIOM <- function(file, ...) {
     .require_package("biomformat")
     biom <- biomformat::read_biom(file)
-    makeTreeSEFromBiom(biom, ...)
+    convertFromBIOM(biom,...)
 }
 
-#' @rdname makeTreeSEFromBiom
-#'
-#' @param x object of type \code{\link[biomformat:read_biom]{biom}}
-#'
-#' @export
+#' \code{convertFromBIOM} creates a TreeSE object from a BIOM object.
 #' @importFrom S4Vectors make_zero_col_DFrame DataFrame
 #' @importFrom dplyr %>% bind_rows
-makeTreeSEFromBiom <- function(
-        x, prefix.rm = removeTaxaPrefixes, 
+#' @rdname convert
+#' @export
+convertFromBIOM <- function(
+        obj, prefix.rm = removeTaxaPrefixes, 
         removeTaxaPrefixes = FALSE, rank.from.prefix = rankFromPrefix, 
         rankFromPrefix = FALSE,
         artifact.rm = remove.artifacts, remove.artifacts = FALSE, ...){
     # input check
     .require_package("biomformat")
-    if(!is(x,"biom")){
```

Comment:
The name of object is changed from obj to x indicating that your branch is out of sync